fixtures/cloudflare-dev-module-resolution/entry-workerd.ts

```diff
@@ -0,0 +1,35 @@
+export default {
+  async fetch(_request: Request, env: any) {
+    return Response.json({
+      ...(await getReactValues()),
+      ...(await getRemixRunCloudflareValues()),
+    });
+  },
+};
+
+async function getReactValues() {
+  try {
+    const { default: React, version: ReactVersion } = await import('react');
+    return {
+      'typeof React': typeof React,
+      'typeof React.cloneElement': typeof React.cloneElement,
+      reactVersionsMatch: React.version === ReactVersion,
+    };
+  } catch {
+    return {};
+  }
+}
+
+async function getRemixRunCloudflareValues() {
+  try {
+    const { json, createCookie } = await import('@remix-run/cloudflare');
+    return {
+      'type of remix cloudflare json({})': typeof json({}),
```

Comment:
Sorry, pedantic but let's make this consistent with `getReactValues`:

```suggestion
      'typeof remix cloudflare json({})': typeof json({}),
```